### PR TITLE
add archive and unarchive tests

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

**Comment by rafaelfolco** [comment]
> https://github.com/k-rister/crucible-ci/blob/dev-kmr/.github/actions/integration-tests/run-ci-stage1#L323-L329
> 
> 3 of those 4 tests will do far less if this were merged...thoughts on how to effectively test the new and the old at the same time?

why far less?

**Comment by k-rister** [comment]
> > https://github.com/k-rister/crucible-ci/blob/dev-kmr/.github/actions/integration-tests/run-ci-stage1#L323-L329
> > 3 of those 4 tests will do far less if this were merged...thoughts on how to effectively test the new and the old at the same time?
> 
> why far less?

If we archive the results of each test after it occurs, when we get to the end of all the tests and run these commands there won't be any results for them to process.  So it will only test the code paths associated with not processing any results as opposed to actively testing the code paths that run when there are results present -- which is probably the code paths we are most concerned about.

**Comment by k-rister** [comment]
I started working on an enhancement to archive/unarchive yesterday just because I thought it would be nice to have, but now I'm thinking it will make testing it easier as well.  See https://github.com/perftool-incubator/crucible/pull/208

Once that is merged I can fixup this PR to process all the results at the end of a run using shell wildcards.  It will defer the space savings until the end of the run but I don't think that is a big deal given other recent improvements (https://github.com/perftool-incubator/crucible-ci/commit/7a59ce3cd3f09cdbb64f91b395e5cbfbac9ddad2).

File: .github/actions/integration-tests/run-ci-stage1 [hunk 1]
```diff
@@ -316,6 +316,12 @@ run_cmd "crucible es init"
 
 run_cmd "crucible es rebuild"
 
+for scenario in $(echo "${CI_SCENARIOS}" | sed -e "s/,/ /g"); do
+    run_cmd "crucible archive /var/lib/crucible/run/${scenario}*"
+
```
Comment (rafaelfolco) [inline .github/actions/integration-tests/run-ci-stage1:321]: blank line: alert of COD

**Review by rafaelfolco — APPROVED** [review]

**Pull request merged**